### PR TITLE
Add UI design for calculator

### PR DESCRIPTION
**Pull request by ghermaico135**
- [ ] Add UI design for calculator

### CODE REVIEW

File: src/Components/Style.css [hunk 1]
```diff
@@ -0,0 +1,73 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  /* min-width: 1200px; */
```
Comment (rustam-tolipov) [inline src/Components/Style.css]: **You have done a good job!**

- [ ] Your app has some cleanup jobs to complete. For starting, please remove this commented code!

File: src/Components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        <div className="grid-item" id="item-six">6</div>
+        <div className="grid-item grid-item-color" id="item-sub">-</div>
+        <div className="grid-item" id="item-one">1</div>
+        <div className="grid-item" id="item-two">2</div>
+        <div className="grid-item" id="item-three">3</div>
+        <div className="grid-item grid-item-color" id="item-add">+</div>
+        <div className="grid-item-zero grid-item" id="item-zero">0</div>
+        <div className="grid-item" id="item-dot">.</div>
+        <div className="grid-item grid-item-color" id="item-equal">=</div>
+      </div>
```
Comment (rustam-tolipov) [inline src/Components/Calculator.js:29]: - [ ] Kindly remove these mentioned files from the project file! Also, make sure your app is working as expected after removing them!

Screenshot 2023-06-13 at 19 19 30

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hello @ghermaico135,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_
_Good readme_
_Well-implemented code_

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Check the comments under the review._

You can also consider:
N/A

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by amejid — APPROVED** [review]
Hi @ghermaico135, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Linter checks passing :heavy_check_mark: 
- Correct workflow :heavy_check_mark: 
- Good UI :heavy_check_mark: 
- Boilerplate files removed :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**